test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.33
+2.33-1
```

Comment:
this is needed to rebuild the test image with this change to the client.  I don't know if this should be done separately or if it's ok to include here.  @claudiubelu do you know? 

I've tested this with an image: `jsturtevant/agnhost:2.33-1-windows-amd64-1809`